pkgs/applications/misc/organicmaps/default.nix

```
@@ -0,0 +1,89 @@
+{ lib
+, mkDerivation
+, fetchFromGitHub
+, fetchpatch
+, cmake
+, ninja
+, pkg-config
+, which
+, python3
+, makeWrapper
+, qtbase
+, qtsvg
+, libGLU
+, zlib
+}:
+
+mkDerivation rec {
+  pname = "organicmaps";
+  version = "2021-05-15";
```

```suggestion
  version = "unstable-2021-05-15";
```
Github says this is still a pre-release

And if you set this, consider also setting the `meta.mainProgram` to the main executable of this derivation, because of a bug making the unstable part of the `pname`.

---

The docs say:

> If a package is not a release but a commit from a repository, then the version part of the name must be the date of that (fetched) commit. The date must be in "YYYY-MM-DD" format. Also append "unstable" to the name - e.g., "pkgname-unstable-2014-09-23".

And I think a prerelease is still a release, not a commit. There are other instances of this in nixpkgs (grep for "beta"). Is there a pre/suffix for prereleases?

I set mainProgram anyway since the executable is named OMaps.

---

> Is there a pre/suffix for prereleases?

No, the docs are incorrect here.

---

@SuperSandro2000 Could you clarify? Do you think it should be marked as unstable?

---

If they use this version format upstream it does not need unstable.